forge-gui/res/cardsfolder/upcoming/tazri_stalwart_survivor.txt

```diff
@@ -0,0 +1,12 @@
+Name:Tazri, Stalwart Survivor
+ManaCost:2 W
+Types:Legendary Creature Human Warrior
+PT:3/3
+S:Mode$ Continuous | Affected$ Creature.YouCtrl | AddAbility$ Mana | Description$ Each creature you control has "{T}:  Add one mana of any of this creature's colors. Spend this mana only to activate an ability of a creature. Activate only if this creature has another activated ability."
+SVar:Mana:AB$ ManaReflected | Cost$ T | Valid$ Defined.Self | ColorOrType$ Color | ReflectProperty$ Is | RestrictValid$ Activated.Creature+inZoneBattlefield | IsPresent$ Card.Self+hasOtherActivatedAbility | SpellDescription$ Add one mana of any of this creature's colors. Spend this mana only to activate an ability of a creature. Activate only if this creature has another activated ability.
+A:AB$ Mill | Cost$ W U R B G | NumCards$ 5 | RememberMilled$ True | SubAbility$ DBChangeZone | SpellDescription$ Mill five cards.
+SVar:DBChangeZone:DB$ ChangeZoneAll | ChangeType$ Creature.IsRemembered+hasNonManaActivatedAbility | Origin$ Graveyard | Destination$ Hand | SubAbility$ DBCleanup | SpellDescription$ Put all creature cards with activated abilities that aren't mana abilities from among the milled cards into your hand.
```

Comment:
We usually add Exile because of rulings like this:
> If a replacement effect causes a player to exile the top four cards of their library instead of putting them into their graveyard as Gyruda’s triggered ability resolves, the creature card you choose may be one of those cards in exile.

Reply:
Ah, because they're still technically "milled" regardless of zone they go into...

Hmm, might `Origin$ All` be better futureproofing then?

Reply:
I'd have to look up the rule but I think it only works with public zones

Reply:
Fair enough... so maybe a bigger refactor later if needed.